examples/secure-boot/build-current-images.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+# Copyright 2024 Google LLC and contributors
```

Comment:
2025?

Reply:
I started writing it in 2024, and I've been advised to only use the earliest date, and not include additional years that the file was touched.